scripts/lbnl_fs.nhc

```diff
@@ -564,3 +575,64 @@ function check_fs_iused() {
     done
     return 0
 }
+
+# https://stackoverflow.com/questions/14366390/check-if-an-element-is-present-in-a-bash-array
+array_contains () {
+    local seeking=$1; shift
```

Comment:
why shift?

Reply:
I just stole it, ask the author :)

Reply:
It sets up the array to become the first argument, allowing the `for element; do ...`